package.json

```diff
@@ -450,10 +450,10 @@
     },
     "devDependencies": {
         "@types/highlight.js": "^9.12.4",
-        "@types/image-size": "^0.7.0",
+        "@types/image-size": "^0.8.0",
         "@types/markdown-it": "^10.0.2",
         "@types/mocha": "^5.2.6",
-        "@types/node": "^12.12.58",
+        "@types/node": "^14.10.1",
```

Comment:
* You told me that [new `image-size` breaks the compatibility](https://github.com/yzhang-gh/vscode-markdown/commit/9b7fb3a5aa672847b288c6f87822dafb5b14ad6b#commitcomment-42278369).
* Due to **webpack** and **VS Code**, we have to stick to **Node.js 12** in the foreseeable future.

Thus, updating these two type definitions looks not a good idea.

Reply:
`image-size@0.8.0` looks fine (`0.9.1` doesn't).

> * Due to **webpack** and **VS Code**, we have to stick to **Node.js 12** in the foreseeable future.

I agree.